behandlingsprosess/src/main/java/no/nav/foreldrepenger/behandling/steg/avklarfakta/fp/KontrollerFaktaRevurderingStegImpl.java

```diff
@@ -375,12 +370,11 @@ private void kopierResultaterAvhengigAvStartpunkt(Behandling revurdering,
         kopierOpptjeningVedBehov(origBehandling, revurdering);
 
         if (StartpunktType.BEREGNING_FORESLÅ.equals(revurdering.getStartpunkt())) {
-            beregningsgrunnlagKopierOgLagreTjeneste.kopierResultatForGRegulering(finnBehandlingSomHarKjørtBeregning(origBehandling).getId(),
-                    revurdering.getId(), stp.getFørsteUttaksdatoGrunnbeløp());
+            beregningTjeneste.kopier(BehandlingReferanse.fra(revurdering), BehandlingReferanse.fra(origBehandling), BehandlingStegType.FORESLÅ_BEREGNINGSGRUNNLAG);
         }
 
         if (StartpunktType.UTTAKSVILKÅR.equals(revurdering.getStartpunkt()) || StartpunktType.TILKJENT_YTELSE.equals(revurdering.getStartpunkt())) {
-            beregningTjeneste.kopier(BehandlingReferanse.fra(revurdering), BehandlingReferanse.fra(origBehandling), BeregningsgrunnlagTilstand.FASTSATT);
+            beregningTjeneste.kopier(BehandlingReferanse.fra(revurdering), BehandlingReferanse.fra(origBehandling), BehandlingStegType.FASTSETT_BEREGNINGSGRUNNLAG);
```

Comment:
Vi bør antagelig ikke drøye for lenge med å skille kopierHeleBG fra kopierFramTilSteg
Ellers kommer noen til å klø seg i hodet om noen år